lib/bootloader_setup.pm

```diff
@@ -56,6 +58,91 @@ sub stop_grub_timeout {
     send_key 'up';
 }
 
+=head2 add_custom_grub_entries
+
+Add custom grub entries with extra kernel parameters.
+It adds 3rd line with default options + 4th line with advanced options.
+Extra kernel parameters are taken in GRUB_PARAM variable.
+
+e.g.  grub entries before:
+* SLES 15
+* Advanced options for SLES 15
+* Start bootloader from a read-only snapshot
+
+grub entries with GRUB_PARAM='ima_policy=tcb' and calling add_custom_grub_entries:
+* SLES 15
+* Advanced options for SLES 15
+* SLES 15 (ima_policy=tcb)
+* Advanced options for SLES 15 (ima_policy=tcb)
+* Start bootloader from a read-only snapshot
+
+And of course the new entries have ima_policy=tcb added to kernel parameters.
+
+=cut
```

Comment:
IMHO you should use a bit more POD formatting in your documentation, e.g. wrap the parameters in `C<>`. All not wrapped content should be plain english, `GRUP_PARAM` is certainly not :)

Reply:
Thanks, I'll fix it.